### ISSUE TITLE
Unable to install on ubuntu

### ISSUE THREAD

**Issue by dwonghy** [RestyaPlatform/board#4394]
[restyaboard_install.log](https://github.com/RestyaPlatform/board/files/8251406/restyaboard_install.log)
unable to re-install restyaboard after uninstalling, please see attached log file

Please advice, thanks!

**Comment by sridhar391** [comment]
@dwonghy sorry for your inconvenience, we are checking the issue and we will update you as soon as possible

**Comment by sridhar391** [comment]
@dwonghy  We have checked the log file and there seems to be an Postgresql updation issue, please tell us if you have tried to install the postgreql manually?

**Comment by dwonghy** [comment]
@sridhar391 I dont think so, I only followed the instruction in the shell script. Is there any way to fix this problem?
